core/src/test/java/org/jboss/pnc/build/finder/core/CompletedBuildTest.java

```diff
@@ -51,11 +51,13 @@ void verifyCompletedBuilds() throws KojiClientException, MalformedURLException {
         Checksum checksum1 = new Checksum(
                 ChecksumType.md5,
                 "46148535be98c75c900837ecea491c71",
-                "hibernate-validator-6.0.10.Final-redhat-1.pom");
+                "hibernate-validator-6.0.10.Final-redhat-1.pom",
+                10L);
```

Comment:
I don't really like the `10L` being used as a fake size for everything. On the other hand, it would take time to look up the real sizes. Maybe if there were such a thing as unknown size like `-1L`. but I'm not sure.

Reply:
I'll see what I can do!